interface/fax/fax_dispatch.php

```diff
@@ -615,43 +614,43 @@ function allCheckboxes(issel) {
 
 <body class="body_top" onunload='imclosing()'>
 
-<center><h2><?php echo xlt('Dispatch Received Document'); ?></h2></center>
+<h2 class='text-center'><?php echo xlt('Dispatch Received Document'); ?></h2>
 
 <form method='post' name='theform'
  action='fax_dispatch.php?<?php echo ($mode == 'fax') ? 'file' : 'scan'; ?>=<?php echo attr_url($filename); ?>&csrf_token_form=<?php echo attr_url(CsrfUtils::collectCsrfToken()); ?>' onsubmit='return validate()'>
 <input type="hidden" name="csrf_token_form" value="<?php echo attr(CsrfUtils::collectCsrfToken()); ?>" />
 
 <p><input type='checkbox' name='form_cb_copy' value='1'
- onclick='return divclick(this,"div_copy");' />
-<b><?php echo xlt('Copy Pages to Patient Chart'); ?></b></p>
+onclick='return divclick(this,"div_copy");' />
+<strong><?php echo xlt('Copy Pages to Patient Chart'); ?></strong></p>
```

Comment:
The `<p>` element should have the `font-weight-bold` class and the `<strong>` tag can then be removed